walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -10,8 +10,10 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
       <div id="quote-container"></div>
+      <button onclick="getGreeting()">Fetch a greeting</button>
```

Comment:
Do you want this to be part of your change?

This is part of the week2 examples, and I don't think you have a getGreeting() method defined here, so I don't think this works.  I think this was just a mistake when we were making modifications to the portfolio, right?

If so, undo the change, and then recommit it.